cmd/print_diff.go

```diff
@@ -0,0 +1,19 @@
+package cmd
```

Comment:
i think it's ok to have it in cmd package, but let's make it an object, like Diff which would have Print method.